src/sentry/api/endpoints/sentry_app_publish_request.py

```diff
@@ -43,7 +44,7 @@ def post(self, request, sentry_app):
             ):
                 return Response(
                     {
-                        "detail": "Must upload a black and white logo for issue linking integrations."
+                        "detail": "Must upload a black icon for issue and stack trace linking integrations."
```

Comment:
`black icon` doesn't make much sense to me. I think `black and white icon` is clearer

Reply:
I specifically put black because we do validation that the icon only contains black or transparent pixels, white isn't allowed.

Reply:
I guess maybe for this error message specifically it doesn't need to mention the color, maybe just "Must upload an icon for issue and stack trace linking integrations" ? What do you think?